R/pkg/R/generics.R

```diff
@@ -387,6 +387,16 @@ setGeneric("value", function(bcast) { standardGeneric("value") })
 #' @export
 setGeneric("agg", function (x, ...) { standardGeneric("agg") })
 
+#' alias
+#'
+#' Set a new name for a Column or a SparkDataFrame. Equivalent to SQL "AS" keyword.
+#'
+#' @name alias
+#' @rdname alias
+#' @param object x a Column or a SparkDataFrame
+#' @param data new name to use
```

Comment:
On the bright side it looks like matching `@rdname` and `@aliases` like:

```r
#' alias
#'
#' @aliases alias,SparkDataFrame-method
#' @family SparkDataFrame functions
#' @rdname alias,SparkDataFrame-method
#' @name alias
...
```
and

```r
#' alias
#'
#' @aliases alias,SparkDataFrame-method
#' @family SparkDataFrame functions
#' @rdname alias,SparkDataFrame-method
#' @name alias
...
```
(I hope this is what you mean) indeed solves SPARK-18825. But it doesn't generate any docs for these two and makes CRAN checker unhappy:

```
Undocumented S4 methods:
  generic 'alias' and siglist 'Column'
  generic 'alias' and siglist 'SparkDataFrame'
```
Docs for generic are created but it doesn't help us here. Even if we bring `@examples` there we still have to deal with CRAN.

Theres is also my favorite `\name must exist and be unique in Rd files` which doesn't gives us much room here, does it?

I opened to suggestions, but personally I am out ideas. I've been digging trough `roxygen` docs, but between CRAN,  S4 requirements, `roxygen` limitation and our own rules there is not much room left.